From 49786b27ebf3fb2a8792973505367febf3406482 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Mon, 22 Mar 2021 18:28:06 +0000
Subject: GCC: fix type confusion in EffectPaintPropertyNode

GCC gets confused if a method and an alias have same name. This can be
avoided by adding explicit namespace.

Bug: 819294
Change-Id: I992544e792f7f3f77d69f9ac282886c79c4f758b
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2773011
Reviewed-by: Chris Harrelson <chrishtr@chromium.org>
Commit-Queue: Stephan Hartmann <stha09@googlemail.com>
Cr-Commit-Position: refs/heads/master@{#865264}
---
 .../platform/graphics/paint/effect_paint_property_node.h      | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/third_party/blink/renderer/platform/graphics/paint/effect_paint_property_node.h b/third_party/blink/renderer/platform/graphics/paint/effect_paint_property_node.h
index fa0995c3d507..516b3f59adb8 100644
--- a/third_party/blink/renderer/platform/graphics/paint/effect_paint_property_node.h
+++ b/third_party/blink/renderer/platform/graphics/paint/effect_paint_property_node.h
@@ -287,8 +287,8 @@ class PLATFORM_EXPORT EffectPaintPropertyNode
     return state_.compositor_element_id;
   }
 
-  const DocumentTransitionSharedElementId& DocumentTransitionSharedElementId()
-      const {
+  const blink::DocumentTransitionSharedElementId&
+  DocumentTransitionSharedElementId() const {
     return state_.document_transition_shared_element_id;
   }
 
-- 
2.25.1

